modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -562,7 +562,7 @@ def _reset(df, *axis_lengths, partition_idx):
                 new_columns = None
 
             return self.__constructor__(
-                self._modin_frame.broadcast_apply_full_axis(
+                self._modin_frame.apply_full_axis(
                     axis=1,
                     func=_reset,
                     other=None,
```

Comment:
```suggestion
```
`apply_full_axis` method doesn't have `other` parameter so we should remove it when calling.